util-hadoop/src/main/java/com/google/cloud/hadoop/util/HadoopCredentialsConfiguration.java

```diff
@@ -81,6 +82,15 @@ public class HadoopCredentialsConfiguration {
   public static final HadoopConfigurationProperty<String> SERVICE_ACCOUNT_JSON_KEYFILE_SUFFIX =
       new HadoopConfigurationProperty<>(".auth.service.account.json.keyfile");
 
+  /**
+   * Key suffix used to configure the path to a JSON file containing a workload identity federation,
+   * i.e. external account credential configuration. Technically, this could be a JSON containing an
+   * service account impersonation url and credential source. but this setting is only used in the
+   * workload identity federation flow and is namespaced as such.
+   */
+  public static final HadoopConfigurationProperty<String> WORKLOAD_IDENTITY_FILE_SUFFIX =
```

Comment:
nit: Please follow the pattern and keep the field name same as the property name?